examples/apigee-x-vpn-hub-and-spoke-design/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.tfstate
+*.tfstate.backup
+.terraform.lock.hcl
+.terraform/
```

Comment:
Please replace the last line with **/.terraform/*
This way providers will not be included in the commit. They are quite big files

This should exclude the lock file as well.

Reply:
Also add these two lines

*.tfstate
*.tfstate.*

To avoid committing tfstate files.